extensions/amp-animation/0.1/web-animations-polyfill.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {installWebAnimations} from 'web-animations-js/web-animations.install';
+
+
+const POLYFILLED = '__AMP_WA';
```

Comment:
Consider a more specific identifier, e.g. `__AMP_WA_POLYFILL`.

Reply:
Noted. I think I'd rather leave the constant that was in place.